lib/errors.js

```diff
@@ -11,9 +181,36 @@ class BadParametersError extends ES6Error {
   constructor (requiredParams, actualParams) {
     super(`Parameters were incorrect. We wanted ` +
           `${JSON.stringify(requiredParams)} and you ` +
-          `sent ${JSON.stringify(actualParams)}`);
-    this.jsonwpCode = 5; // TODO use status library for bad command
+          `sent ${JSON.stringify(actualParams)}`, 13);
   }
 }
 
-export { NotYetImplementedError, BadParametersError };
+const errors = { NotYetImplementedError,
```

Comment:
since we are not `export`ing, lets do 

```js
const errors = {NotYetImplementedError,
                ...
                NoSuchFrameError};
```

i.e., we only add a space in between the braces like this when we are importing/exporting. this is just a javascript object